doc/payum-example/README.md

```diff
@@ -0,0 +1,61 @@
+## Abstract
+
+
+The gateway available in this repository is meant to provide you an integration with stripe [Checkout Server](https://stripe.com/docs/payments/checkout/server).
+
+In order to achieve this, you will have to follow those steps:
+
+
+# Installation
+
+## Composer
+
+```bash
+composer require combodo/stripe-v3
+```
+
+Note: for now I do not plan to follow BC rules, use semantic versioning or other, so please check if the code is working after each upgrade.
+
+## Register the gateway
+
+See the service tagged `payum.gateway_factory_builder` [in the example of conf](./payum-example/app/config/payum.yml).
```

Comment:
the relative paths used in this file will wail : 
./payum-example/path/to/file
should be:
./path/to/file

ie : 
> ...in the example of conf](./app/config/payum.yml)

Reply:
You are totally right, this is my mistake, it's already corrected

Reply:
I'm sorry, I tried to refresh my browser, but I still se many `./payum-example` in this readme

Reply:
This is really weird : https://github.com/Combodo/CombodoPayumStripe/pull/9/commits/baa731104ec02215de23c1b774a0b32dfbb5f29f

Reply:
Here we are :)